cfme/utils/appliance/__init__.py

```diff
@@ -985,10 +993,11 @@ def diagnose_evm_failure(self):
         logger.info('Checking appliance database')
         if not self.db.online:
             # postgres isn't running, try to start it
-            cmd = 'systemctl restart {}-postgresql'.format(self.db.postgres_version)
-            result = self.db.ssh_client.run_command(cmd)
-            if result.failed:
-                return 'postgres failed to start:\n{}'.format(result.output)
+            logger.info('Database is not online, restarting')
+            try:
+                self.db_service.restart()
+            except SystemdException:
+                return 'postgres failed to start'
```

Comment:
Do you thing result isn't necessary in this case ?

Reply:
included logging in latest commit